src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -169,97 +258,98 @@ <h4 class="panel-title border-bottom-small mt" translate>
             <translate>{{'CHANGING_FREQUENCY'}}</translate>
           </loading>
         </div><!-- // panel-body -->
-      </div><!-- // panel -->
-      <div class="panel panel-default give-modal-panel" ng-if="$ctrl.productData.frequency !== 'NA'">
-        <h4 class="panel-title border-bottom-small mt0 pt-" translate>
-          {{'RECURRING_START'}}
-        </h4>
-        <div class="panel-body pt0">
-          <div class="row">
-            <div class="col-sm-3">
-              <div class="form-group">
-                <label>
-                  <span translate>{{'MONTH'}}</span>
-                  <select class="form-control form-control-subtle"
-                          ng-model="$ctrl.itemConfig['RECURRING_START_MONTH']"
-                          ng-options="(m | date:'MM') as (m | date:'MMMM, yyyy') for m in $ctrl.possibleTransactionMonths($ctrl.nextDrawDate)"
-                          ng-change="$ctrl.changeStartDay($ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'])">
-                  </select>
-                </label>
+      </div>
```

Comment:
This change (removal of comment on div) is undone in the next commit, should be squashed.